examples/analytics.amp.html

```diff
@@ -127,6 +127,22 @@
 </script>
 </amp-analytics>
 
+<!-- Krux tracking. Ensure you replace CONFID with your configuration ID. -->
+<amp-analytics type="krux" config="https://cdn.krxd.net/controltag/amp/CONFID.json">
```

Comment:
I think the best thing would be if you could have a barebones config that you point to here. It could be hosted in the same folder as this examples file.